src/memory/allocator.hpp

```diff
@@ -181,6 +181,37 @@ namespace impl {
             }
         };
 
+        // bare bones implementation of standard compliant allocator for managed memory
+        template <size_type Alignment=1024>
```

Comment:
The `alignment` member returns `Alignement`. I think that the method you are referring to is the one on the `device_allocator` below the `managed_allocator`.

To be honest, this is a bit messy, and needs one hell of a refactoring. It is pretty simple to abuse this interface and violate the alignment guarentees, but that is a job for another PR.